### PR TITLE
fixes up-sample script

### PR DESCRIPTION
**Pull request by vniche**
**- What I did**

- updated the postgresql helm parameter (line 29)
- updated secret key reference (line 33)
- changed the approach on waiting for postgres to be ready and create analytics database (lines 39 to 42)
- added secret for confidential client (line 52)

**- How to verify it**

same old `make up-sample`

**- Description for the changelog**
fixes the script to deploy an local development environment

### CODE REVIEW

**Comment by nathanmartinszup** [comment]
Hi @vniche.

In order for your pull request to be accepted, your [commits must be signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits). You can check how to do this in our [contribution guide](https://github.com/ZupIT/horusec-operator/blob/main/CONTRIBUTING.md#developer-certificate-of-origin---dco).

Thank you very much for the contribution!

File: config/samples/sample_install_dependencies.sh [hunk 1]
```diff
@@ -26,29 +26,20 @@ helm repo update
...
+helm install postgresql --set auth.database=horusec_db bitnami/postgresql
 
 echo "Getting user and pwd of dependencies..."
 export POSTGRES_USERNAME="postgres"
-export POSTGRES_PASSWORD=$(kubectl get secret postgresql -o jsonpath="{.data.postgresql-password}" | base64 --decode)
+export POSTGRES_PASSWORD=$(kubectl get secret postgresql -o jsonpath="{.data.postgres-password}" | base64 --decode)
 export RABBITMQ_USERNAME="user"
 export RABBITMQ_PASSWORD=$(kubectl get secret rabbitmq -o jsonpath="{.data.rabbitmq-password}" | base64 --decode)
 export JWT_SECRET="4ff42f67-5929-fc52-65f1-3afc77ad86d5"
+export CONFIDENTIAL_CLIENT_SECRET="4ff42f67-5929-fc52-65f1-3afc77ad86d5"
```
Comment (nathanmartinszup) [inline config/samples/sample_install_dependencies.sh:37]: Why is this necessary?

**Comment by vniche** [comment]
> Hi @vniche.
> 
> In order for your pull request to be accepted, your [commits must be signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits). You can check how to do this in our [contribution guide](https://github.com/ZupIT/horusec-operator/blob/main/CONTRIBUTING.md#developer-certificate-of-origin---dco).
> 
> Thank you very much for the contribution!

Thanks again @nathanmartinszup, just fixed 👊

**Comment by nathanmartinszup** [comment]
Thank you very much @vniche. 

As for the Keycloak values, I believe they are not necessary, as this installation will only use the standard Horusec authentication, not the Keycloak.

**Review by wiliansilvazup — APPROVED** [review]

**Comment by wiliansilvazup** [comment]
> Thank you very much @vniche.
> 
> As for the Keycloak values, I believe they are not necessary, as this installation will only use the standard Horusec authentication, not the Keycloak.

@nathanmartinszup I changed this in my PR #158 :)

**Comment by wiliansilvazup** [comment]
Thank you very much @vniche for your contribution!

You rock :rocket:
